root/account/Donation.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {l} from '../static/scripts/common/i18n';
+import Frag from '../components/Frag';
+import StatusPage from '../components/StatusPage';
+
+type Props = {|
+  +contactURL: string,
+  +days: number,
+  +nag: boolean,
+|};
+
+const Donation = ({contactURL, days, nag}: Props) => (
+  <StatusPage title={l('Donation Check')}>
+    {nag
+      ? (
+        <Frag>
+          <p>
+            {l('We have not received a donation from you recently. If you have just made a PayPal donation, then we have not received a notification from PayPal yet. Please wait a few minutes and reload this page to check again.')}
+          </p>
+          <p>
+            {l('If you would like to make a donation, {donate|you can do that here}. If you have donated, but you are still being nagged, please {contact|contact us}.',
+              {__react: 'true', contact: contactURL, donate: 'https://metabrainz.org/donate'})}
```

Comment:
Maybe we should add `DONATE_URL` to constants.js too? It's also used in root/layout/components/Footer.js

Reply:
Done with https://github.com/metabrainz/musicbrainz-server/pull/699/commits/0d4d014f6ab9fccef6852d9ffd4f3e16a09b8260. Same question as for `CONTACT_URL`.